src/styles/LoginStyles.ts

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
```

Comment:
I moved the styling from your Login file to this separate file that uses styled-components!

This will be the styling convention that we will move forward with to preserve higher readibility/organization.